php/utils.php

```diff
@@ -215,7 +215,7 @@ function handle_exception($exception) {
  * @return [boolean] $is_hermes True if Terminus is operating on Hermes
  */
 function is_hermes() {
-  $is_hermes = (TERMINUS_HOST == 'dashboard.getpantheon.com');
```

Comment:
I think I've cleaned all those up and the tests still pass. But ben noticed that the is_hermes() function doesn't seem to be used anywhere, so we might as well get rid of it if we don't need it.